plutus-benchmark/bls12-381-costs/test/Spec.hs

```diff
@@ -0,0 +1,13 @@
+module Main where
+
+import PlutusBenchmark.BLS12_381.RunTests (runTests)
+import PlutusBenchmark.Common (goldenVsTextualOutput)
+
+outputFile :: String
+outputFile = "bls12-381-costs.txt"
+
+goldenFile :: FilePath
+goldenFile = "bls12-381-costs/test/bls12-381-costs.golden"
```

Comment:
I don't like having this relative path here but I don't think I can get rid of it.  If we change the name of the directory and fail to change this then the test will still succeed but write a new golden file into the wrong directory (which will be created if it doesn't exist).  Something similar will happen if you type `cabal run bls12-381-costs-test` (__not__ `cabal test .,.,.`) anywhere but in the root of `plutus-core` (but then you shouldn't use `cabal run` to run tests).

Reply:
> You could add a check that the file already exists before running the test.

Yes, that might be a sensible idea. The way that cabal and tasty-golden deal with these things is maybe a little annoying (tasty-golden does report when it creates a new golden file, but it's easy to miss because it's in a pale colour).  Once we've generated the golden file for the first time it should be safe to assume that it should exist for ever afterwards. Now I'm wondering what we do with all of the golden files that we already have.